lib/sidekiq-status/web.rb

```diff
@@ -48,7 +48,17 @@ def has_sort_by?(value)
         jids = namespace_jids.map{|id_namespace| id_namespace.split(':').last }
         @statuses = []
 
-        jids.each do |jid|
+        if params[:max_items] == 'all'
+          @max_items = 'all' if params[:max_items] == 'all'
+          max_items = jids.size if @max_items == 'all'
+        else
+          is_a_valid_max_item_param = %w(20 50 100).include?(params[:max_items])
```

Comment:
I would like to make the max items array configurable, probably as a writeable attribute on `Sidekiq::Status::Web` to allow someone to change the defaults in their own setup.